Formula/lib/libtirpc.rb

```diff
@@ -13,23 +13,20 @@ class Libtirpc < Formula
   depends_on :linux
 
   def install
-    system "./configure",
-      "--disable-dependency-tracking",
-      "--disable-silent-rules",
-      "--prefix=#{prefix}"
+    system "./configure", "--disable-silent-rules", *std_configure_args.reject { |s| s["--disable-debug"] }
     system "make", "install"
   end
 
   test do
     (testpath/"test.c").write <<~EOS
-      #include <rpc/des_crypt.h>
+      #include <rpc/rpc.h>
       #include <stdio.h>
-      int main () {
-        char key[] = "My8digitkey1234";
-        if (sizeof(key) != 16)
-          return 1;
-        des_setparity(key);
-        printf("%lu\\n", sizeof(key));
+
+      int main() {
```

Comment:
I might be missing something, but I think this no longer tests anything from `libtirpc`? We probably need to link in + call at least one symbol here.